### ISSUE TITLE
The issue of loss function.

### ISSUE THREAD

**Issue by mliwang** [sunilkmaurya/GNN_Ranking#1]
I would like to ask if the actual centrality is used when optimizing the model, if computing resources and time have been consumed to calculate the centrality, what is the significance of your work with GNN?

**Comment by sunilkmaurya** [comment]
Hi! Great question! You are right that we need to calculate the centrality value of few graphs to create a training dataset.
The idea is once you train the model, then for any number of given input graphs you can get approximate node ranking almost immediately (inference time is low). While in case of traditional methods, you would need to calculate centrality values
for all the graphs, which is computationally expensive and time taking. Also our approach is not unique in this way. Several
other methods work in similar way like approximate graph edit distance [1], predicting chemical properties of molecules [2],
more recently physics simulation [3] using GNNs and many more, use computational data to train the model.

In addition, we train our model on smaller graphs and then predict the node ranking on much larger graphs. We provide more
details in our paper.

[1] https://arxiv.org/abs/1808.05689
[2] https://arxiv.org/abs/1704.01212
[3] https://arxiv.org/abs/2010.03409
